docs/develop/node/community/node-community-updates.md

```diff
@@ -0,0 +1,32 @@
+---
+id: node-community-updates
+title: Node Community Updates
+sidebar_label: Node Community Updates
+---
+
+## NEAR Validator Community Channels
+
+NEAR Protocol will communicate with validators using these channels:
+
+1. New releases, technical support and bug tracking in the Github repository of [`nearcore`](https://github.com/near/nearcore/issues) and Github project [`Node Experience`](https://github.com/orgs/near/projects/18).
```

Comment:
let's separate new releases from technical support and bug tracking to be more explicit